technical-reference/grid-properties.md

```diff
@@ -143,6 +143,19 @@ Provides a value for the <strong>version</strong> attribute of the XSL <strong>o
 <br />
 Provides a value for the <strong>encoding</strong> attribute of the XSL <strong>output</strong> element.</td>
 </tr>
+<tr class="odd">
+<td><a href="xslt-transform-engine-grid-property.md">XSLT transform engine</a></td>
+<td>Compiler</td>
+<td>Indicates XSLT transform engine to be used in runtime transformation.<br />
+</td>
+</tr>
+<tr class="even">
+<td><a href="use-xsl-transform-grid-property.md">Use XSL Transform</a></td>
+<td>Compiler</td>
+<td>Indicates whether to use legacy .Net <a href="https://docs.microsoft.com/en-us/dotnet/api/system.xml.xsl.xsltransform">XslTransform</a> to achieve XSLT transformation, otherwise <a href="https://docs.microsoft.com/en-us/dotnet/api/system.xml.xsl.xslcompiledtransform">XslCompiledTransform</a> will be used if "False" is selected.<br />
+<br />
+This option will only be used when ".Net Framework" is selected as "XSLT transform engine".</td>
+</tr>
```

Comment:
The above are showing to be 1 new row in the table. 
There needs to be two different rows - one for XSLT transform engine and the other for Use XSL Transform

Reply:
I believe it is only a problem in the preview of github website. My code indicate it should be 2 different rows, and it is displayed well in my local preview.